README

```diff
@@ -274,3 +274,101 @@ refresh			Attempts to reinsert all modules that are currently
+- can contained **path** -  path to the object (target) the process attempted to access.
+
+The next step is to analyze the received log and find the corresponding **te** file by **scontext**, for example,
+for **dnsmasq_t** there is a corresponding **dnsmasq.te** file and in this file need to add a rule that
+allows **scontext** to access **tcontext**.
+
+### Recommended steps for editing refpolicy.
+
+To find the required **te** file where to add the rule need to find a line declaring the **scontext** type
+(label) how in example `type dnsmasq_t;`. Next, after the **Local policy comment**, add the following rule.
+Sometimes the rules are logically grouped, for example, there are rules for working with the `/etc` directory, etc.
+
+**It is important** that before adding a rule, need to look for a possible macro for **tcontext** declared in a
+file with the **if** extension, to find the necessary file **if**, first need to find the corresponding **te** file
+in which the label will be declared and then take the **if** file with the same name. For example, the
+`syslogd_runtime_t` label is declared in the `logging.te` file, then the macro should be looked for in the
+`logging.if` file.
+
+If some file or directory marked with an incorrect label or `unlabeled_t`, then  need to find the **te**
+file where the label of interest is declared and take the file with the **fc** extension and declare the context for
+the label of interest there. For example
+```
+/usr/bin/aos_vis  --  gen_context(system_u:object_r:aos_exec_t,s0)
+```
+which means that the `aos_vis` file will be labeled `aos_exec_t`, this is an executable process.
+
+To see what would be the correct or possible label in the `unlabeled_t`, need to run the `fixfiles -F -f relabel`
+command or another command, after that the resource or process's will be labeled with proper context.
+
+**It is important** if need to create a new label, then must be create three files **te, fc, if** with the name
+of the label that with which is created. At the same time, the **te** file in which declare a new label type must not
+be empty, the **fc, if** files can be empty.
+
+Sometimes in the logs contains denials security policies, but which may not be very critical, for example, get the
+file attribute or read a file that may not affect the system, for example, when login into the system, being created
+policies to denial write to the system log
+```
+avc:  denied  { write } for  pid=1034 comm="login" name="dev-log" dev="tmpfs" ino=20945
+scontext=system_u:system_r:local_login_t:s0-s0:c0.c1023
+tcontext=system_u:object_r:syslogd_runtime_t:s0 tclass=sock_file permissive=0
+```
+For these denials **not need to add rules** to the refpolicy if it is not necessary.
+
+After the corresponding security policy is added, it is necessary to build the refpolicy again through yocto.
+To find interest a custom module with the **pp** extension and install it on the system without reinstalling
```

Comment:
Unclear sentence, please rephrase.

Reply:
Removed this sentence as it is incorrect in our context